go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -232,7 +232,12 @@ func (agent *ActionAgent) startReplication(ctx context.Context, pos mysql.Positi
 	defer remoteCancel()
 	posStr, err := tmc.MasterPosition(remoteCtx, ti.Tablet)
 	if err != nil {
-		return vterrors.Wrap(err, "can't get master replication position")
+		// It is possible that though MasterAlias is set, the master tablet is unreachable
+		// Log a warning and let tablet restore in that case
+		// If we had instead considered this fatal, all tablets would crash-loop
```

Comment:
Can we change one of the e2e test cases to take the master down before restoring one of the tablets? Would that have caught this?

Reply:
I was able to reproduce with a unit test, and verified that the fix works.